acceptance/suites/tests/555-puppetserver-service/reboot.rb

```diff
@@ -0,0 +1,55 @@
+if (options[:type] == 'pe') then
+  pssn='pe-puppetserver'
```

Comment:
I'm not a huge fan of this variable name, but please don't change it unless you're already back in the file making other changes.

My reasoning is that `pssn` doesn't mean anything to me without having to go back through and fully understand the context around where the variable was bound.  Only then does pssn make sense as the puppetserver service name.

A better name would avoid making the reader figure out the acronym, so maybe something like `service_name` as a suggestion.